Formula/naga-cli.rb

```diff
@@ -0,0 +1,152 @@
+class NagaCli < Formula
+  desc "Shader translation command-line tool"
+  homepage "https://github.com/gfx-rs/naga"
+  license any_of: ["Apache-2.0", "MIT"]
+  head "https://github.com/gfx-rs/naga.git", branch: "master"
+
+  stable do
+    url "https://github.com/gfx-rs/naga/archive/refs/tags/v0.9.0.tar.gz"
+    sha256 "5d31b90774b1bb7ef77056911d2a49014ebe4863a6faa83f696f208bbbf3ee02"
+
+    # patch required to fix the version number on the naga-cli crate
+    patch do
+      url "https://github.com/gfx-rs/naga/commit/350171ed2f45612386cc91a0ee9ff79d918939a9.patch?full_index=1"
+      sha256 "45d8d6f0c799cc6ebf198c76dbf991889d4483ba401b8c12b9cdd425a627fa19"
+    end
+  end
+
+  depends_on "rust" => :build
+
+  conflicts_with "naga", because: "naga-cli also ships a naga binary"
```

Comment:
We can add the `conflicts_with` after this is merged so we don't need to rebuild `naga` too.